CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+## v1.1.0 (11/12/2019)
+
+**Common**
+* The deprecated `cancel` method in `CancellationToken` removed.
+* The curl network implementation fixed to compile on 32 bits architecture.
+* The `disk_path` field in `CacheSettings` is deprecated. The `disk_path_mutable` field should be used instead.
+* A new synchronous `CallApi` method was added to `OlpClient`.
+* A bug is fixed when cmake was not able to check that the `pipe2` symbol exists.
+
+**olp-cpp-sdk-authentication**
+* The `SignInClient` method in `AuthenticationClient` is deprecated in favor of the newly introduced `SignInClient` method with a different signature.
+* A `scope` support was added through the `SignInProperties` struct.
```

Comment:
It is not very obvious what scope support is.
Maybe "Added scope support in OAuth2 trough `olp::authentication::SignInProperties`. Users are now able to access project bound resources by using the `olp::authentication::SignInProperties::scope` field."